spending.csv

```diff
@@ -0,0 +1,18 @@
+Food,2022-11-13,30.0

```

Comment:
let's remove csv file. I want to test it by myself

Reply:
it means to give user access to read his .csv?